### PR TITLE
Add tests & linting

### PR DESCRIPTION
**Pull request by kraynel**

### CODE REVIEW

File: generators/loopback/templates/server/.eslintrc [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "plugins": [
+    "react"
```
Comment (tcheymol) [inline generators/loopback/templates/server/.eslintrc:26]: @kraynel  Why using React plugin on a Loopback app ?

File: generators/loopback/templates/server/package.json [hunk 2]
```diff
@@ -3,15 +3,16 @@
...
+    "migrate:up": "db-migrate up -e $NODE_ENV",
     "posttest": "npm run lint && nsp check",
-    "start": " npm run migrate:up && yarn stop && ./node_modules/.bin/pm2 start pm2.yml",
-    "start:prod": "npm run migrate:up -- -e production && NODE_ENV=production ./node_modules/.bin/pm2 start pm2.yml",
-    "start:watch": "npm run migrate:up && ./node_modules/.bin/pm2 start server/server.js --watch ./server",
-    "stop": "./node_modules/.bin/pm2 kill"
+    "start": " npm run migrate:up && yarn stop && pm2 start pm2.yml",
+    "start:prod": "npm run migrate:up -- -e production && NODE_ENV=production pm2 start pm2.yml",
+    "start:watch": "npm run migrate:up && pm2 start server/server.js --watch ./server",
+    "stop": "pm2 kill",
```
Comment (tcheymol) [inline generators/loopback/templates/server/package.json:14]: @kraynel Why using only globally installed packages ?

File: generators/loopback/templates/server/.eslintrc [hunk 3]
```diff
@@ -1,3 +1,51 @@
...
+    "no-underscore-dangle": 0,
+    "no-constant-condition": 0,
+    "no-duplicate-imports": 0,
+    "comma-dangle": 0,
+    "import/no-unresolved": 0,
+    "import/no-extraneous-dependencies": 0,
+    "import/extensions": 0,
+    "react/jsx-boolean-value": 0,
+    "react/jsx-filename-extension": 0,
+    "react/no-unused-prop-types": 0,
```
Comment (tcheymol) [inline generators/loopback/templates/server/.eslintrc:47]: @kraynel Idem, a lot of react here

File: generators/loopback/templates/server/package.json (hunk 2 above)
Comment (kraynel) [reply generators/loopback/templates/server/package.json:14]: The `node_modules/.bin/` directory is automatically added to the path when writing npm scripts, so you don't need to prefix all commands by this path.

**Pull request closed without merging**